iceoryx_posh/cmake/iceoryx_posh_deployment.cmake

```diff
@@ -22,7 +22,7 @@ if(ONE_TO_MANY_ONLY)
 endif()
 
 if(NOT IOX_COMMUNICATION_POLICY)
-    message(STATUS "[i] Using m:n communication only!")
+    message(STATUS "[i] Using m:n communication!")
```

Comment:
I have seen this in the other MR before. Maybe we should merge it to see the real delta.

Reply:
That makes sense. If you want a sneak preview, before #852 is merged, you can use the branch compare feature of GitHub to only see the delta.